openapi/index.yaml

```diff
@@ -347,6 +347,36 @@ paths:
           description: Server Error
           schema:
             $ref: '#/definitions/ProblemJson'
+  /notify-login:
+    post:
+      operationId: startNotifyLoginProcess
+      description: >-
+        Initiate the Orchestrator to send an email to the user regarding a
+        recently performed login.
+      tags:
+        - restricted
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: '#/definitions/UserLoginParams'
+      responses:
+        '202':
+          description: Request accepted
+        '400':
+          description: Invalid request.
+          schema:
+            $ref: '#/definitions/ProblemJson'
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not found
+        '429':
```

Comment:
How this can happen? This function should start the orchestrator without using the database and any rate limit is present on the HTTP layer. Am I missing something?

Reply:
You're right, removed in [[#IOPID-383] remove unused status code](https://github.com/pagopa/io-functions-app/pull/282/commits/a839f6105a173de3f4a6e8373d71b84b7fda6749)